src/script/lib/modals/customTransaction/customTransaction.html

```diff
@@ -21,8 +21,8 @@ <h3 class="modal-title" translate>customTransactionHeadline</h3>
                    placeholder="0.42"
                    ng-model="transactionValue" required autocomplete="off" autofocus>
         </div>
-        <br>
-        <div class="input-group input-group-lg">
+        <br ng-if="enableComments">
+        <div class="input-group input-group-lg" ng-if="enableComments">
```

Comment:
@schinken I just noticed that this change broke the comment feature. For some reason the comment value is no longer passed to `submitTransaction(transactionValue, transactionComment)`

I will check for a solution...